index.md

```diff
@@ -72,3 +72,81 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
+
+In this hypothetical scenario, the UN’s International Climate Change Authority (UNFCCC) establishes a semi-automatic certification and verification system for creating official, digital UN Renewable Energy Certificates or REC NFTs.
+
+HydroElec Inc., has recently upgraded a dam to support the production of hydroelectric power and is going through the process of claiming 100 tons of Carbon Credits from the UN’s International Climate Change Authority (UNFCCC). By using the UN’s new system for digital RECs, HydroElec is able to reliably monetize the environmental impact of creating clean electricity with minimal bureaucratic overhead.
+## Scenario
+The UNFCCC defines a class template for the REC2022 NFT and registers this token on the ixo blockchain. They also certify Certification Agents and Claim Verification Agents, issuing these agents Verifiable Credentials. Providing these entities with the proper capabilities and Verifiable Credentials to prove their qualifications.
+
+HydroElec starts a new project and creates a formal proposal that meets the UNFCCC requirements. This proposal is digitally signed by HydroElec and submitted to the UNFCCC. The UNFCCC forwards the proposal to an assigned Certifier who reviews the proposal and, finding it meritorious, issues a Certification Credential, which they give to HydroElect.
+
+After certification, HydroElec can now produce verifiable credentials attesting to the production of clean energy according to the proposal. Once the system is operational, they contract a UN Claim Verifier who visits the project site, inspects the capacity, measures current production, and evaluates the Verifiable Credentials created by HydroElect’s turbines. Finding everything in order, The verifier issues their own findings in a Verifiable Credential, which is given to HydroElec.
+
+Once they have all the necessary credentials HydroElect submits everything to the ixo NFT module (where the UNFCCC registered their token), which triggers the minting of official UNFCCC Renewable Energy NFTs, issued directly to HydroElec. HydroElec submits regularly to transform his hydro plant’s output into NFTs on a regular basis. They can take these NFTs to any market and exchange them for digital or fiat currency, as enabled by the markets.
+
+## Distinctions
+
+This use case presents a complex set of capabilities for automatiing the verification of real world impact. The token module used by the UN must be able to support the ability to set sophisticated constraints on the minting of NFTs by otherwise unknown parties: it isn't the UN that mints the NFTs, it is the REC producer, who mints the NFTs according to specific policies set by the UN. These policies must be programmatically specifiable and verifiable, so that once set up, the operational creation of NFTs no longer requires UN interaction.
+## Artifacts
+
+A list of key technical artifacts used to realize the use case:
+
+### Verifiable Credentials
```

Comment:
These aren't explicitly named in the scenario text. Not sure if that matters

Reply:
@jandrieu Your thoughts on Will's comment?